### PR TITLE
Update to Questrade.IQEdge 6.6.56.172

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Resolve #101823 
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103167)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie What happen to the Azure Pipelines?

**Comment by BrandonWanHuanSheng** [comment]
I also try that to install without any warning before submitting.

**Comment by BrandonWanHuanSheng** [comment]
It look like it quite annoying.

**Comment by BrandonWanHuanSheng** [comment]
> Hello @BrandonWanHuanSheng,
> 
> During installation testing, this application failed to install without user input. Did you forget to add Silent or SilentWithProgress switches?
> 
> This can also happen when a dependency is missing. You can test with https://github.com/microsoft/winget-pkgs#test-your-manifest.
> 
> Please investigate a fix and resubmit the Pull Request.
> 
> You may also try using the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) to determine the proper installer type. If the installer type is MSIX, MSI, or a known installer technology like NullSoft, Inno, etc. the wingetcreate tool can detect them and the winget client will know what switches to pass. If it's a .exe installer of an unknown type, you will need to search to determine the proper switches for Silent and SilentWithProgress.
> 
> Template: msftbot/validationError/installation/unattended

It might failed again to this error.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie It looks like this error is keep popping up.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie  It look like what is Validation-Unattended-Failed

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2023-04-22 16:36:33.466   CurStepChanged raised an exception (fatal). 
> 2023-04-22 16:36:33.466   Exception message: Runtime Error (at 5:366): Internal error: Cannot access 64-bit registry keys on this version of Windows. 

InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230423001657

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

This `x86` package is odd - it errors on 32-bit OS and warns on 64-bit OS. Should we remove it?

File: manifests/q/Questrade/IQEdge/6.6.56.172/Questrade.IQEdge.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+InstallModes:
+- silent
+UpgradeBehavior: install
+InstallerSwitches: 
+ Silent: /Slient
+ Custom: /NoRestart
+Installers:
+- Architecture: x86
+  InstallerUrl: https://media.questrade.com/iq_downloads/QuestradeIQEdge.exe
+  InstallerSha256: 85030F7C8DE66EC82883C417A39C79C97B582B4323B784BAC247BA6038F94FF4
```
Comment (stephengillie) [inline manifests/q/Questrade/IQEdge/6.6.56.172/Questrade.IQEdge.installer.yaml:20]:
```suggestion
```

**Comment by BrandonWanHuanSheng** [comment]
>  This `x86` package is odd - it errors on 32-bit OS and warns on 64-bit OS. Should we remove it?

I already removed it from the pipelines.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**